packages/zoe/test/unitTests/contracts/loan/test-addCollateral.js

```diff
@@ -0,0 +1,83 @@
+// ts-check
+
+import '../../../../exported';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import '@agoric/install-ses';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import test from 'ava';
+
+import { makeAddCollateralInvitation } from '../../../../src/contracts/loan/addCollateral';
+import { makeFakePriceAuthority } from '../../../fakePriceAuthority';
+import buildManualTimer from '../../../../tools/manualTimer';
+
+import {
+  setupLoanUnitTest,
+  makeSeatKit,
+  performAddCollateral,
+} from './helpers';
+
+test.todo('makeAddCollateralInvitation - test bad proposal');
```

Comment:
additionally:
 * can we verify that a new liquidation trigger was rescheduled?

Reply:
Yes, that is tested already.

Reply:
See 'borrow, then addCollateral, then getLiquidationPromise'

Reply:
That tests ends with `  t.falsy(liquidated);`, so I think it shows that neither was triggered. I'm interested in the case where both price levels are exceeded, but only the rescheduled one should cause liquidation. 

value is  at 12
borrow 100 with 16 of collateral.   (12 * 16 > 1.5 * 100)
value goes to 10   (10 * 16 > 1.5 * 100)
addCollateral of 10 
value goes to 8; with 16 collateral, would have liquidated     (8 * 16 < 1.5 * 100)
value goes to 5; now we should actually liquidate.   (5 * 26 < 1.5 * 100)

Reply:
Oh, good point. That `liquidated` is old - the liquidation code doesn't call that anymore. Let me revise that test.

Reply:
Liquidation is added to that test